helpers/custom_helpers.rb

```diff
@@ -38,6 +38,10 @@ def add_visible_class(path)
     end
   end
 
+  def progress_bar(language)
+    "#{language.lessons.complete*5}%"
+  end
+
```

Comment:
This method needs to be refactored. We should:

- [ ] Improve the naming to more accurately describe what it does (It only returns a percentage as a string, not a whole progress bar)
- [ ] Decouple from data structure. (Only pass in what you need, not the whole object.)
- [ ] Coerce argument to integer in order to guard against nil